tests/validations.sh

```diff
@@ -190,8 +192,12 @@ pipeline_exists $AZDO_ORG_URL $AZDO_PROJECT $FrontEnd
 spk service install-build-pipeline -o $AZDO_ORG -r $mono_repo_dir -u $remote_repo_url -d $AZDO_PROJECT -l $services_dir -p $ACCESS_TOKEN_SECRET -v $FrontEnd  >> $TEST_WORKSPACE/log.txt
 
 # Verify the pipeline was created
-pipeline_created=$(az pipelines show --name $FrontEnd-pipeline --org $AZDO_ORG_URL --p $AZDO_PROJECT)
-# TODO: Verify the pipeline run was successful
+pipeline_name="$FrontEnd-pipeline"
```

Comment:
We can pass this name into line 192 with `-n` so that we can always be sure the name will match.

Reply:
@mtarng addressed